docs/content/how-to/agent/create-agent-cluster.md

```diff
@@ -135,72 +226,87 @@ spec:
 
 ~~~sh
-$ oc get agents -n ${HOSTED_CONTROL_PLANE_NAMESPACE}
+oc -n ${HOSTED_CONTROL_PLANE_NAMESPACE} get agents
+
 NAME                                   CLUSTER   APPROVED   ROLE          STAGE
 86f7ac75-4fc4-4b36-8130-40fa12602218                        auto-assign
 e57a637f-745b-496e-971d-1abbf03341ba                        auto-assign
+~~~
+
+~~~sh
+oc -n ${HOSTED_CONTROL_PLANE_NAMESPACE} patch agent 86f7ac75-4fc4-4b36-8130-40fa12602218 -p '{"spec":{"installation_disk_id":"/dev/sda","approved":true,"hostname":"worker-0.example.krnl.es"}}' --type merge
 
-$ oc patch agent 86f7ac75-4fc4-4b36-8130-40fa12602218 -p '{"spec":{"installation_disk_id":"/dev/sda","approved":true,"hostname":"worker-0.example.krnl.es"}}' --type merge
-$ oc patch agent 23d0c614-2caa-43f5-b7d3-0b3564688baa -p '{"spec":{"installation_disk_id":"/dev/sda","approved":true,"hostname":"worker-1.example.krnl.es"}}' --type merge
+oc -n ${HOSTED_CONTROL_PLANE_NAMESPACE} patch agent 23d0c614-2caa-43f5-b7d3-0b3564688baa -p '{"spec":{"installation_disk_id":"/dev/sda","approved":true,"hostname":"worker-1.example.krnl.es"}}' --type merge
+~~~
+
+~~~sh
+oc -n ${HOSTED_CONTROL_PLANE_NAMESPACE} get agents
 
-$ oc get agents -n ${HOSTED_CONTROL_PLANE_NAMESPACE}
 NAME                                   CLUSTER   APPROVED   ROLE          STAGE
 86f7ac75-4fc4-4b36-8130-40fa12602218             true       auto-assign
 e57a637f-745b-496e-971d-1abbf03341ba             true       auto-assign
 ~~~
 
-### metal3
+### Metal3
 
-We will leverage the Assisted Service and Hive to create the custom ISO as well as the Baremetal Operator to perform the
-installation.
+We will leverage the Assisted Service and Hive to create the custom ISO as well as the Baremetal Operator to perform the installation.
 
-* Enable the Baremetal Operator to watch all namespaces as the `baremetalhost` object for the hosted cluster will be created in the
-  `${HOSTED_CONTROL_PLANE_NAMESPACE}` namespace:
+> **WARN:** Since the `BaremetalHost` objects will be created outside the baremetal-operator namespace we need to configure the operator to watch all namespaces.
 
 ~~~sh
 oc patch provisioning provisioning-configuration --type merge -p '{"spec":{"watchAllNamespaces": true }}'
 ~~~
 
-> **NOTE**: This will trigger a restart of the `metal3` pod in the `openshift-machine-api` namespace.
+> **INFO:** This will trigger a restart of the `metal3` pod in the `openshift-machine-api` namespace.
 
 * Wait until the `metal3` pod is ready again:
 
 ~~~sh
 until oc wait -n openshift-machine-api $(oc get pods -n openshift-machine-api -l baremetal.openshift.io/cluster-baremetal-operator=metal3-state -o name) --for condition=containersready --timeout 10s >/dev/null 2>&1 ; do sleep 1 ; done
 ~~~
 
-* Set the variables required for the BMC details of the worker that is going to be added:
+Now, we can go ahead and create our BaremetalHost objects, we need to configure some variables required for being able to boot our bare-metal nodes.
```

Comment:
I would say:

"Now we can go ahead and create our BaremetalHost objects.  We will need to configure some variables required to be able to boot our bare-metal nodes."

Reply:
Fixed.